rfcs/0046-platform-support-tiers.md

```diff
@@ -80,6 +84,8 @@ Platform-specific things for arbitrary packages should not be too complicated
 Support: native bootstrap tools are available, cross-build toolchains in the
 binary cache, partial tooling support
 
+Native stdenv is a channel-blocking job
```

Comment:
this words is a bit confusing because there is a heavily bitrotted "impure" native stdenv.

Reply:
Argh, indeed.